tests/jballerina-unit-test/src/test/resources/test-src/statements/ifelse/type-guard.bal

```diff
@@ -367,7 +367,8 @@ type Person record {
 };
 
 type Student record {
-    *Person;
```

Comment:
Do we need this change to test file?

Reply:
There is a L1 identified with this - #15780